spotui/src/DeviceMenu.py

```diff
@@ -38,6 +38,21 @@ def restart(self):
 
     def __map_devices(self, item):
         available_space = self.endx - self.startx - 6
+        
+        #injection of current device
+        status = self.api.get_playing()
```

Comment:
We'll always have self.device_id set on MainForm when opening the device menu so there's no need to make another request here - we can just provide device_id as a constructor argument here and use that instead.